File: src/main/java/com/beemer/fromis9server/introduction/model/Members.java
package com.beemer.fromis9server.introduction.model;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Data;

import java.time.LocalDate;

@Data
@Entity
@Table(name = "Members")
public class Members {
    @Id
    @Column(name = "name", nullable = false)
    private String name;

    @Column(name = "imageUrl", nullable = false)
    private String imageUrl;

    @Column(name = "birth", nullable = false)
    private LocalDate birth;

    @Column(name = "position", nullable = false)
    private String position;

    @Column(name = "blood", nullable = false)
    private String blood;
}

File: src/main/kotlin/com/beemer/unofficial/fromis9/fromis9/entity/Members.kt
package com.beemer.unofficial.fromis9.fromis9.entity

import jakarta.persistence.*
import java.util.Date

@Entity
@Table(name = "Members")
data class Members(
    @Id
    @Column(name = "name", nullable = false)
    var name: String,

    @Column(name = "birth", nullable = false)
    var birth: Date,

    @Column(name = "profile_image", nullable = false)
    var profileImage: String,

    @OneToOne(mappedBy = "member", cascade = [CascadeType.ALL])
    var details: MemberDetails? = null
)
